.rubocop.yml

```diff
@@ -8,8 +8,8 @@ AllCops:
     - 'dashboard/scripts/**/*'
     - 'dashboard/db/**/*'
     - 'dashboard/db/*'
-  RunRailsCops: true
   DisplayCopNames: true
+  TargetRubyVersion: 2.2
```

Comment:
Great question!  I should've included this in the PR description.

My update process was:
1. Update the rubocop version [specified in the `Gemfile`](https://github.com/code-dot-org/code-dot-org/pull/6858/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R160).
2. Run `bundle update --source rubocop` to update `Gemfile.lock`.
3. Import the new `.rubocop_hound.yml` from Hound's repo.
4. Run `bundle exec rubocop --auto-gen-config --exclude-limit 0` to regenerate `.rubocop_todo.yml` (the new version of rubocop has a different set of rules).
5. Run `rake lint:ruby` to verify, and fix up the two `config.ru` files with lint errors.

The new lint errors appeared because the old config excluded these files.